troposphere/static/theme/theme.json.j2

```diff
@@ -1,4 +1,11 @@
 {
+    "images": {
+        {% if USE_THEME_IMAGES -%}
+        "imagesDir": "/theme/themeImages"
+        {%- else -%}
+        "imagesDir": "/theme/themeImagesDefault"
+        {%- endif %}
+    },
```

Comment:
This is more of a note for consideration ... 

If we bring `imagesDir` into the theme.json file, we might be able to remove the definition of this value from [`./themeImagesPath.js.j2`](https://github.com/cyverse/troposphere/blob/zesty-zapdos/themeImagesPath.js.j2).